### PR TITLE
Add criteria for community resources

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**
Adds suggested message so community resources cannot be mistaken for official parts of the Design System.
Fixes https://github.com/alphagov/design-system-team-internal/issues/430

### CODE REVIEW

File: src/community/contribution-criteria/index.md.njk [hunk 1]
```diff
@@ -123,3 +122,7 @@ Before linking from the Design System to a [community resource or tool](/communi
 - development resources output the same markup as GOV.UK Frontend
 - it is not being charged for or used to promote a private business
 - it's compatible with the [Service Standard](https://www.gov.uk/service-manual/service-standard) and [Technology code of practice](https://www.gov.uk/government/publications/technology-code-of-practice/technology-code-of-practice)
+
+A community resource or tool should not be mistaken for an official part of the Design System. To help users decide whether to use your resource or tool and know how to get support for it, add this message to the resource README and any website links:
```
Comment (calvin-lau-sig7) [inline src/community/contribution-criteria/index.md.njk:126]:
```suggestion
A community resource or tool should not be mistaken for an official part of the Design System. 

To help users decide whether to use your resource or tool and know how to get support for it, add this message to the resource README and any website links:
```

File: src/community/contribution-criteria/index.md.njk [hunk 2]
```diff
@@ -123,3 +122,7 @@ Before linking from the Design System to a [community resource or tool](/communi
...
+
+>‘Name of resource or tool’ is a community ‘resource/tool’ of the GOV.UK Design System. The Design System team is not responsible for it and cannot support you with using it. Contact ‘contributor’ directly if you need help or you want to request a feature.
```
Comment (calvin-lau-sig7) [inline src/community/contribution-criteria/index.md.njk:128]:
```suggestion
[Name of resource or tool] is a community [resource/tool] of the GOV.UK Design System. The Design System team is not responsible for it and cannot support you with using it. Contact [contributor] directly if you need help or you want to request a feature.
```

**Review by calvin-lau-sig7 — COMMENTED** [review]
Formats example text to match style used in Service Manual.

File: src/community/contribution-criteria/index.md.njk [hunk 3]
```diff
@@ -114,7 +114,6 @@ Before a new component or pattern is published the working group reviews the imp
 Before linking from the Design System to a [community resource or tool](/community/resources-and-tools) the Design System team will review it to make sure that:
 
 - it has a clear owner
```
Comment (calvin-lau-sig7) [inline src/community/contribution-criteria/index.md.njk:116]:
```suggestion
- it has a clear owner
- it cannot be mistaken for an official resource
```

File: src/community/contribution-criteria/index.md.njk [hunk 4]
```diff
@@ -123,3 +124,9 @@ Before linking from the Design System to a [community resource or tool](/communi
 - development resources output the same markup as GOV.UK Frontend
 - it is not being charged for or used to promote a private business
 - it's compatible with the [Service Standard](https://www.gov.uk/service-manual/service-standard) and [Technology code of practice](https://www.gov.uk/government/publications/technology-code-of-practice/technology-code-of-practice)
+
+To help users decide whether to use your resource or tool and tell them how to get support for it, add this message to the resource README and any website links:
```
Comment (calvin-lau-sig7) [inline src/community/contribution-criteria/index.md.njk:128]:
```suggestion
To help users decide whether to use your resource or tool and tell them how to get support for it, add this message to the resource README and after any website links:
```

**Review by timpaul — APPROVED** [review]

**Pull request merged**